net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/User.xml

```diff
@@ -76,6 +76,31 @@
                 <sessionlimit_max_session_limit type="IntegerField">
                     <Required>N</Required>
                 </sessionlimit_max_session_limit>
+                <servicetype type="OptionField">
+                    <Required>N</Required>
+                    <multiple>Y</multiple>
+                    <OptionValues>
+                        <Option1 value="1">1</Option1>
```

Comment:
I would make these more user-friendly, but for now that is robust enough. Thanks!

Reply:
(Display Text value with numeric value attached, like: THIS-SETTING (1) )

Reply:
Indeed, I thought that this work might be a good start for new community members which insist on more descriptive values :)